src/rust/aggregator/src/pipelines/order_history_pipelines.rs

```diff
@@ -0,0 +1,598 @@
+use std::{collections::HashMap, sync::Arc};
+
+use bigdecimal::{BigDecimal, Zero};
+use chrono::{DateTime, Duration, Utc};
+use sqlx::{PgConnection, PgPool, Transaction};
+
+use aggregator::{util::*, Pipeline, PipelineAggregationResult, PipelineError};
+use sqlx_postgres::Postgres;
+use tokio::sync::RwLock;
+
+use crate::MAX_BATCH_SIZE;
+
+pub const TIMEOUT: std::time::Duration = std::time::Duration::from_secs(1);
+
+const BPSS_TIMES_TEN: [i32;8] = [0, 25, 50, 100, 250, 500, 1000, 2000];
+
```

Comment:
How about a comment like?

`Basis point amounts multiplied by 10, to enable 2.5 bps indexing.`

If that is clarified up here at the top then there shouldn't be too much confusion later on